modules/authentication/proc-enabling-authentication-with-github.adoc

```diff
@@ -1,151 +1,181 @@
+You can use these secrets in the {product-short} configuration files by using their respective environment variable name.
 +
-`AUTH_GITHUB_APP_ID`:: Enter the saved **App ID**.
-`AUTH_GITHUB_CLIENT_ID`:: Enter the saved **Client ID**.
-//`GITHUB_HOST_DOMAIN`:: Enter your GitHub host domain: `github.com` unless you are using GitHub Enterprise.
-`GITHUB_ORGANIZATION`:: Enter your GitHub organization name, such as `__<your_github_organization_name>__'.
-`GITHUB_ORG_URL`:: Enter `$GITHUB_HOST_DOMAIN/$GITHUB_ORGANIZATION`.
-`GITHUB_CLIENT_SECRET`:: Enter the saved **Client Secret**.
-`GITHUB_PRIVATE_KEY_FILE`:: Enter the saved **Private key**.
-`GITHUB_WEBHOOK_URL`:: Enter your {product-short} URL: `pass:c,a,q[{my-product-url}]`.
-`GITHUB_WEBHOOK_SECRET`:: Enter the saved *Webhook secret*.
-
-. . To set up the GitHub authentication provider and enable integration with the GitHub API in your {product-short} custom configuration, edit your custom {product-short} config map such as `{my-app-config-config-map}`, and add the following lines to the `{my-app-config-file}` file content:
+`AUTHENTICATION_GITHUB_CLIENT_ID`::
+Enter the saved **Client ID**.
+
+`AUTHENTICATION_GITHUB_CLIENT_SECRET`::
+Enter the saved **Client Secret**.
+
+`AUTHENTICATION_GITHUB_HOST_DOMAIN`::
+Enter the GitHub host domain: `github.com`.
+
+`AUTHENTICATION_GITHUB_ORGANIZATION`::
+Enter your GitHub organization name, such as `__<your_github_organization_name>__`.
+
+. Enable the GitHub organization provisioning plugin (`backstage-plugin-catalog-backend-module-github-org`).
+This plugin ingests GitHub users and groups to the {product-short} software catalog.
 +
-.`{my-app-config-file}` file fragment with mandatory fields to enable authentication with GitHub
-[source,yaml]
+.`dynamic-plugins.yaml` file fragment
+[code,yaml]
 ----
-auth:
-  environment: production # <1>
-  providers:
-    github:
-      production:
-        clientId: ${AUTH_GITHUB_CLIENT_ID} # <2>
-        clientSecret: ${AUTH_GITHUB_CLIENT_SECRET}
-integrations:
-  github:
-    - host: ${GITHUB_HOST_DOMAIN}
-      apps:
-        - appId: ${AUTH_GITHUB_APP_ID}
-          clientId: ${AUTH_GITHUB_CLIENT_ID}
-          clientSecret: ${GITHUB_CLIENT_SECRET}
-          webhookUrl: ${GITHUB_WEBHOOK_URL}
-          webhookSecret: ${GITHUB_WEBHOOK_SECRET}
-          privateKey: |
-            ${GITHUB_PRIVATE_KEY_FILE}
-signInPage: github # <3>
+plugins:
+  - package: './dynamic-plugins/dist/backstage-plugin-catalog-backend-module-github-org'
+    disabled: false
 ----
-<1> Mark the environment as `production` and disable the Guest login option in the {product-short} login page.
-<2> Apply the GitHub credentials configured in your {product-short} secrets.
-<3> To enable the GitHub provider as your {product-short} sign-in provider.
-
-.. Optional: Consider adding the following optional fields:
++
+The GitHub authentication plugin (`auth-backend-module-github-provider`) is already enabled by default.
```

Comment:
```suggestion
```
I don't think we need to mention this, this is a core Backstage feature and cannot be disabled.